Task3_Position_the_Cubes/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="index.css">

```

Comment:
The href attribute in the link tag should specify the correct path to the CSS file.